### ISSUE TITLE
Update reference to WCAG1.0

### ISSUE THREAD

**Issue by LJWatson** [w3c/specberus#1161]
Rule 1.6.4 still refers to WCAG1.0. It should be updated to refer to WCAG2.1 instead.

**Comment by jennyliang220** [comment]
@LJWatson could you be more clear about Rule 1.6.4? In rules for wcag, https://www.w3.org/pubrules/doc/rules/?profile=REC#wcag, the link is pointing to WCAG 2.1.

**Comment by LJWatson** [comment]
My apologies. I was looking at an old version that came up first in search results, and missed the superceded message at the top.
